winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
 ---
 layout: post 
 title: Content Types in WinUI AvatarView control | Syncfusion
-description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+The AvatarView control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing without the initials, custom image, or group view types.
+
+{% tabs %}
+{% highlight xaml %}
+
+<Grid>
+<syncfusion:SfAvatarView />
+</Grid>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+// Creating an instance of the AvatarView control.
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+Initials content type is used to show initials to the `AvatarView`. When using the `ContentType` as initials, you need to set the initial character using the following properties:
+
+* `InitialsType` - Defines the type of characters to be displayed.  
+* `AvatarName` - Defines the initials to be displayed.
+
+### InitialsType
+
+The `InitialsType` contains the following two types:
+
+* SingleCharacter
+* DoubleCharacter
+
+You must set the `AvatarName` string property for displaying the initials value in the AvatarView.
+
+**SingleCharacter**
+
+The `SingleCharacter` is used for displaying the first character in the string you have set in the `AvatarName` property.
+
```

Comment:
@MadhumithanJayapathi The single character initials type used to display the first character of the AvatarName property value as the avatar.